src/app/components/mnoe-users-list/mno-users-list.coffee

```diff
@@ -32,8 +32,10 @@
           render: (user) ->
             template: """
             <a ui-sref="dashboard.customers.user({userId: user.id})">
-              <div ng-show="user.name && user.surname">{{::user.name}} {{::user.surname}}</div>
-              <div ng-show="!user.name && !user.surname">nc</div>
+              <div> <span ng-show="user.name && user.surname">{{::user.name }} {{::user.surname}} </span>
+              <span ng-show="!user.name && !user.surname">nc</span>
+              <i title="This user is locked" ng-show="user.locked_at" class="fa fa-lock"> </i>
```

Comment:
You should expose and rely on `access_locked?` as the lock expire after a configurable time:
```ruby
      # Verifies whether a user is locked or not.
      def access_locked?
        !!locked_at && !lock_expired?
      end
```
https://github.com/plataformatec/devise/blob/3-stable/lib/devise/models/lockable.rb#L59